prometheus/src/cluster_metrics.rs

```diff
@@ -0,0 +1,57 @@
+use solana_gossip::cluster_info::ClusterInfo;
+use solana_runtime::bank::Bank;
+
+use crate::{
+    token::Lamports,
+    utils::{write_metric, Metric, MetricFamily},
+};
+use std::{io, sync::Arc, time::SystemTime};
+
+pub fn write_cluster_metrics<W: io::Write>(
+    at: SystemTime,
+    bank: &Arc<Bank>,
+    cluster_info: &Arc<ClusterInfo>,
+    out: &mut W,
+) -> io::Result<()> {
+    let identity_pubkey = cluster_info.id();
+    let version = cluster_info
+        .get_node_version(&identity_pubkey)
+        .unwrap_or_default();
+
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "solana_cluster_identity_public_key_info",
+            help: "The current node's identity",
+            type_: "count",
+            metrics: vec![Metric::new(1)
+                .with_label("identity", identity_pubkey.to_string())
+                .at(at)],
+        },
+    )?;
+
+    let identity_balance = Lamports(bank.get_balance(&identity_pubkey));
+    write_metric(
+        out,
+        &MetricFamily {
+            name: "solana_cluster_identity_balance_total",
```

Comment:
```suggestion
            name: "solana_cluster_identity_balance_sol",
```
The `_total` suffix is typically used for counters that count the total number of times something happened since startup. (E.g. SOL spent on votes.) But in this case we have a gauge, it can go up and down.

Also, we should include the unit in the name. See also https://prometheus.io/docs/practices/naming/

Great idea to include this metric by the way!